src/Analysis/Ast/Impl/Specializations/Typing/Types/AnyType.cs

```diff
@@ -24,6 +24,7 @@ internal sealed class AnyType : LocatedMember, IPythonType {
 
         public override PythonMemberType MemberType => PythonMemberType.Class;
         public string Name => "Any";
+        public string QualifiedName => $"{DeclaringModule.Name}:{Name}";
```

Comment:
Can we make it an extension method to `IPythonType`?

Reply:
Yes